src/test/java/be/seeseemelk/mockbukkit/entity/ThrownExpBottleMockTest.java

```diff
@@ -0,0 +1,54 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import be.seeseemelk.mockbukkit.ServerMock;
+import org.bukkit.Material;
+import org.bukkit.entity.EntityType;
+import org.bukkit.inventory.ItemStack;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ThrownExpBottleMockTest
```

Comment:
```suggestion
class ThrownExpBottleMockTest
```

Reply:
Done in the commit [b52e778](https://github.com/MockBukkit/MockBukkit/pull/913/commits/b52e77896fb4860405828817eb0a62da8f82a1ce)